### PR TITLE
Update to contributing-docs for issue #9976

### LINKED ISSUE #9976: [Docs]  Using Docker section of the Documentation to serve docs site locally has wrong commands
### Current State
In the Using Docker section of the documentation of docs to serve the site locally the command is wrong 
currently it is showing `make docker-docs` if you have docker installed in your system 
[https://docs.meshery.io/project/contributing/contributing-docs#using-docker](url)

also the `make docker` command is also not working

### Desired State
It should be `make docs-docker`
the `make docker` command should be replaced with `make docker-build`

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by sangramrath**
**Notes for Reviewers**

This PR fixes #9976 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ This is because Jekyll always considers the exact version of Ruby unlike JavaScr
 So, you need to follow either of the three steps to resolve this problem;
 
 - Install the required Ruby version by using `rvm` or by any means given above
-- Alternatively, if you have Docker installed, then type `make docker-docs` to view the changes
+- Alternatively, if you have Docker installed, then type `make docs-docker` to view the changes
```
Comment (vishalvivekm) [inline docs/pages/project/contributing/contributing-docs.md:183]: ✔️

File: docs/pages/project/contributing/contributing-docs.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ bundle install{% endcapture %}
 
 If you've Docker and `make` installed in your system, then you can serve the site locally
 
-{% capture code_content %}make docker{% endcapture %}
+{% capture code_content %}make docker-build{% endcapture %}
```
Comment (vishalvivekm) [inline docs/pages/project/contributing/contributing-docs.md:165]:
```suggestion
{% capture code_content %}make docker{% endcapture %}
```
Here  it's expected to run the `make docker` command inside `docs` dir.

Comment (sangramrath) [reply docs/pages/project/contributing/contributing-docs.md:165]: @vishalvivekm Yes. I am making some more changes to this file (mostly verbatim) to make instructions more clear. For example, add above this command that this should be run inside the docs folder.

Additional notes:
- The following statement needs clarification, "**_Alternatively, if you have Docker installed, then type `make docs-docker` to view the changes._**". The command essentially runs the docs site, similar to `make docker` command, just that this command would work from outside the _docs_ folder. It's not clear how this resolves the error.

**Comment by iArchitSharma** [comment]
@sangramrath are you still working on it?

**Comment by l5io** [comment]
🚀 Preview for commit 31c2f0fa2afa188ff6f528d306429ffeb666defa at: https://65f7b9f4ba0342057621c261--meshery-docs-preview.netlify.app

**Comment by sangramrath** [comment]
Yes @iArchitSharma.
@vishalvivekm updates have been pushed with the following changes:
- Corrections were done to the docker command
- Many code snippets inside an ordered list were not using the `<pre class="codeblock_pre">...</pre>` method. This was updated
- Sentence restructure, grammar and other proof reading changes.

**Comment by l5io** [comment]
🚀 Preview for commit 6bb2410da56f252772ddc649257ccf220b3a8211 at: https://66156b59faad7c00787b252a--meshery-docs-preview.netlify.app

**Review by vishalvivekm — APPROVED** [review]
Thanks a bunch @sangramrath

**Pull request merged**